package.json

```diff
@@ -42,6 +42,7 @@
     "url": "git://github.com/ampersandjs/ampersand-form-view"
   },
   "scripts": {
+    "validate": "jshint .",
```

Comment:
I think I copied this from ampersand-state which has a `validate` script in its [package.json](https://github.com/AmpersandJS/ampersand-state/blob/master/package.json#L47). It's sometimes nice to be able to run jshint before attempting to commit, so I'd argue for leaving it but I don't have a strong opinion about it either way.